src/Psalm/Internal/Analyzer/Statements/Expression/Call/FunctionCallAnalyzer.php

```diff
@@ -235,6 +243,45 @@ public static function analyze(
             );
```

Comment:
the AfterEveryFunctionCallAnalysisEvent is the plugin hook. I feel like this should stay at the bottom of the Analyzer.